### PR TITLE
{tools}[system/system] code-server v3.7.3

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)

Requires https://github.com/easybuilders/easybuild-easyblocks/pull/2255

### CODE REVIEW

**Comment by ocaisa** [comment]
Test report by @ocaisa
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2255
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
alanc-VirtualBox - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (sandybridge), Python 3.8.5
See https://gist.github.com/710ae49772fe325182fdba5eacc2e2f9 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2255
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alanc-VirtualBox - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (sandybridge), Python 3.8.5
See https://gist.github.com/86b7f04616cc27c1b42f313a40742ad0 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2255
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
generoso - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz, Python 3.6.8
See https://gist.github.com/92a1a38d799ea6281d3d04f407605ad8 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2255
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
generoso - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz, Python 3.6.8
See https://gist.github.com/d1861a7829df81ab04d58eba1ab98693 for a full test report.

File: easybuild/easyconfigs/c/code-server/code-server-3.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+checksums = [
+    {
+        'code-server-%(version)s-linux-amd64.tar.gz': '7a90f3171c9bc6f65266066e35cc34d48a032910c136ea21116d28f3d7214547', 
+        'code-server-%(version)s-linux-arm64.tar.gz': 'ed0e0aa296bc52bdd927cb388f321d5bd03f625316df02e0004652f0bb035dc0',
+    }
+]
+
+install_cmd = "cp -a code-server-%(version)s-linux-%(mapped_arch)s/* %(installdir)s"
+
+sanity_check_paths = {
```
Comment (boegel) [inline easybuild/easyconfigs/c/code-server/code-server-3.7.3.eb:20]: This and `install_cmd` can go into the easyblock too?

Also, we should run something like `code-server --help` as sanity check command?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/code-server/code-server-3.7.3.eb (hunk 1 above)
Comment (ocaisa) [reply easybuild/easyconfigs/c/code-server/code-server-3.7.3.eb]: Done

**Comment by ocaisa** [comment]
Test report by @ocaisa
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2255
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
generoso - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz, Python 3.6.8
See https://gist.github.com/55f5eba16f88e0b469f87714330a3788 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso EB_ARGS="--include-easyblocks-from-pr 2255"

**Comment by Micket** [comment]
@boegelbot please test @ generoso EB_ARGS="--include-easyblocks-from-pr 2255"

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @ocaisa!

**Pull request merged**